packages/solvuu_build/solvuu_build.0.0.1/descr

```diff
@@ -0,0 +1 @@
+Solvuu's build system.
```

Comment:
Maybe you want to add more lines with a short explanation of the advantages/strong points of your build system?